master/pkg/model/experiment.go

```diff
@@ -535,18 +537,21 @@ type CheckpointV1 struct {
 	Framework         string     `db:"framework" json:"framework"`
 	Format            string     `db:"format" json:"format"`
 	DeterminedVersion string     `db:"determined_version" json:"determined_version"`
+	Size              int        `db:"size"`
```

Comment:
should this one also be int64 explicitly? we aren't consistent in mapping bigint -> int64 in the codebase at all though.